amr-wind/overset/OversetOps.cpp

```diff
@@ -331,9 +335,11 @@ void OversetOps::sharpen_nalu_data()
 
         // Average down fluxes across levels for consistency
         for (int lev = nlevels - 1; lev > 0; --lev) {
+            amrex::IntVect rr =
+                geom[lev].Domain().size() / geom[lev - 1].Domain().size();
             amrex::average_down_faces(
-                GetArrOfConstPtrs(fluxes[lev]), fluxes[lev - 1],
-                repo.mesh().refRatio(lev), geom[lev - 1]);
+                GetArrOfConstPtrs(fluxes[lev]), fluxes[lev - 1], rr,
+                geom[lev - 1]);
```

Comment:
these changes were necessary to avoid a segfault. I didn't get the segfault previously because I wasn't testing multilevel cases. However, this now conforms to the way it's done in other parts of our code.

Reply:
hmmm should it actually be `repo.mesh().refRatio(lev-1)` ? I am a little weirded out of doing our own math here if we have ref ratio available.